script.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    alert('Hello,world!');
```

Comment:
This is going to display an alert when the page loads. This was not a requirement of the assignment.